ParkingZoneApp/Areas/Admin/Models/ParkingZone.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace ParkingZoneApp.Areas.Admin.Models
+{
+    [Table("ParkingZone")]
+    public class ParkingZone
```

Comment:
try to move this model in ParkingZoneApp/Models/ParkingZone.cs Models should be in a separate folder or in a separate layer. Models should not depend on anything. https://code-maze.com/onion-architecture-in-aspnetcore/